.mailmap

```diff
@@ -14,163 +18,203 @@
 Samuel Bosch <BoschSamuel@users.noreply.github.com>
 Scott Kelso <scottkelso07@hotmail.com>
-Scott Lawrence <bytbox@gmail.com>
-Sean Dague <sean@dague.net>
+Sebastian Brandhofer <sebastian.brandhofer@gmail.com>
 Seif Mostafa <csseifms@gmail.com>
-Seyon Sivarajah <seyon.sivarajah@cambridgequantum.com>
-Shaohan Hu <shaohan.hu@ibm.com>
-Shaohan Hu <shaohan.hu@ibm.com> <Shaohan.Hu@ibm.com>
+Shaohan Hu <shaohan.hu@ibm.com> <hushaohan@gmail.com>
+Shelly Garion <46566946+ShellyGarion@users.noreply.github.com>
+Shelly Garion <46566946+ShellyGarion@users.noreply.github.com> <shellygarion@users.noreply.github.com>
 Soolu Thomas <soolu.thomas@ibm.com> <soolu.elto@gmail.com>
-Srujan Meesala <srujanm@gmail.com>
-Stefan Woerner <wor@zurich.ibm.com> <stefan@swoerner.de>
 Stefan Woerner <wor@zurich.ibm.com> <41292468+stefan-woerner@users.noreply.github.com>
+Stefan Woerner <wor@zurich.ibm.com> <stefan@swoerner.de>
+Stefan Woerner <wor@zurich.ibm.com> <wor@zurich.ibm.com>
+Stephen Wood <40241007+woodsp-ibm@users.noreply.github.com>
 Stephen Wood <40241007+woodsp-ibm@users.noreply.github.com> <woodsp@us.ibm.com>
 Sumit Puri <er.sumitpuri@gmail.com>
-Takashi Imamichi <31178928+t-imamichi@users.noreply.github.com>
+Takashi Imamichi <31178928+t-imamichi@users.noreply.github.com> <imamichi@jp.ibm.com>
+Takashi Imamichi <31178928+t-imamichi@users.noreply.github.com> <t.imamichi@gmail.com>
 Tharrmashastha SAPV <51048173+TharrmashasthaPV@users.noreply.github.com>
-Ninad Sathaye <sathayen@users.noreply.github.com>
+Tharrmashastha SAPV <51048173+TharrmashasthaPV@users.noreply.github.com> <tharrmashasthapv@users.noreply.github.com>
+Thomas Alexander <talexander@ibm.com> <thomas.a.alexander@ibm.com>
 Thomas Alexander <talexander@ibm.com> <thomasalexander2718@gmail.com>
 Toshinari Itoko <itoko@jp.ibm.com>
-Viktor Prutyanov <viktor.prutyanov@phystech.edu>
+Toshinari Itoko <itoko@jp.ibm.com> <15028342+itoko@users.noreply.github.com>
+Victor Villar <59838221+vvilpas@users.noreply.github.com> <vvilpas@gmail.com>
+Vishnu Ajith <vishnuajith@gmail.com>
 Vismai Khanderao <59114226+Vismai-Khanderao@users.noreply.github.com>
 Vojtech Havlicek <Vojtech.Havlicek@ibm.com>
-Wei Hu <weihu@cs.virginia.edu>
-Wes Turner <wes.turner@gmail.com>
 Will Bang <will.h.bang@gmail.com>
 Yael Ben-Haim <yaelbh@il.ibm.com>
-Yukio Siraichi <sir.yukio@gmail.com>
-Yunho Maeng <yunho.maeng.dc@gmail.com>
-Yunong Shi <synisnot@gmail.com>
-Sebastian Brandhofer <sebastian.brandhofer@gmail.com>
-Animesh N Dangwal <animesh.leo@gmail.com>
-Faisal Debouni <faisal@megoapps.com>
-Juan Luis Sánchez Toural <toural@es.ibm.com>
 yang.luh <34480044+LuhangYang@users.noreply.github.com>
-Yotam Vaknin <Yotam.Vaknin@Ibm.com> <52409900+yotamvakninibm@users.noreply.github.com>
-Yotam Vaknin <Yotam.Vaknin@Ibm.com> <yotamvak@lnx-quantum.openstack.haifa.ibm.com>
-Łukasz Herok <lukasz.herok@gmail.com>
+Yotam Vaknin <Yotam.Vaknin@Ibm.com> <52409900+yotamvakninibm@users.noreply.github.com>
```

Comment:
Can we fix the final line ending?

Reply:
I'll patch the script to always add one.